unblob/processing.py

```diff
@@ -146,6 +149,18 @@ def __init__(
     def process(self):
         logger.debug("Processing file", path=self.task.path, size=self.size)
 
+        detect = magic.detect_from_filename(self.task.path)
+        skip = any(
+            [detect.name.startswith(pattern) for pattern in self.config.skip_magic]
+        )
```

Comment:
You can get rid of the list comprehension and make the `any` call short-circuiting that way:
```suggestion
        skip = any(
            detect.name.startswith(pattern) for pattern in self.config.skip_magic
        )
```

```
>>> def is_true():
...     print("It is True!")
...     return True
... 
>>> any([is_true() for _ in range(3)])
It is True!
It is True!
It is True!
True
>>> any(is_true() for _ in range(3))
It is True!
True
>>> 
```

Reply:
good point :)